Question 1: Probability that the test will be positive:
0.98 * 0.005

In [1]:
0.98 * 0.005

0.0049

This is for those who have the desease.  However, for those who don't, that is 99.5% of the population, the possibility is 10%
This translates into:

Possibility of those who have it + possibility of those who don't:


In [2]:
0.98 * 0.005 + 0.995 * 0.1

0.1044

Possibility that the result is positive: 10.44%

Question 2: Correctly diagnose a sufferer of Thripshaw's
This is already given, that is 98%

Question 3: Correctly identify a non-sufferer of Thripshaw's

If the test has a posibility of giving a false positive of 10% when given to a non-sufferer, then the posibility of providing a negative when given to a non-sufferer is 100 - 10 = 90%

Question 4: Misclassify the person:
As the test has a probability of giving a good result, 98%, when given to a sufferer, this yields a 2% chance a giving a negative result to a sufferer.  At the same time, 10% chance of giving a false positive to a non-sufferer.  This yields:
1 - (0.002 * 0.005 + 0.1)

In [4]:
0.002 * 0.005 + 0.1

0.10001